src/Illuminate/Http/ApiResponseTrait.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Illuminate\Http;
+
+use InvalidArgumentException;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Pagination\AbstractPaginator;
+
+trait ApiResponseTrait
```

Comment:
Could we call this something else? Like `ApiResponse` or `RespondsToApiCall` or something without the `Trait` suffix..